htsget.md

```diff
@@ -37,31 +37,53 @@ Requests to the retrieval API endpoint may be authenticated by means of an OAuth
 
 ## Errors
 
-Non-successful invocations of the API return an HTTP error code, and the response body contains a JSON object (content-type application/json) with the following structure:
+The server MUST respond with an appropriate HTTP status code (4xx or 5xx) when
+an error condition is detected.  In the case of transient server errors, 
+(e.g., 503 and other 5xx status codes), the client SHOULD implement appropriate
+retry logic as specified in [RFC2616&sect;10.5](https://tools.ietf.org/html/rfc2616#section-10.5).
 
-```json
-{
-   "error": {
-       "type": "NotFound",
-       "message": "No such accession"
-   }
-}
-```
+For errors that are specific to the `htsget` protocol, the response body SHOULD be a JSON object
+(content-type `application/json`) providing machine-readable information about the nature 
+of the error, along with a human-readable description. The structure of this JSON object is 
+described as follows.
+
+### Error Response JSON fields
+
+<table>
+<tr markdown="block"><td>
+`error`  
+_string_
+</td><td>
+The type of error. This SHOULD be chosen from the list below.
+</td></tr>
+<tr markdown="block"><td>
+`message`  
+_string_
+</td><td>
+A message specific to the error providing information on how to debug the problem. Clients 
+MAY display this message to the user.
+</td></tr>
+</table>
```

Comment:
If you can find a way to express nested tables with cells variously containing formatting and multiple paragraphs in any flavour of markdown at all, go for it.  [I don't believe it's possible¹](https://github.com/samtools/hts-specs/pull/180#issuecomment-266769555), so you'd need to either find a way to express this information without using tables or live with the use of HTML for complex tables.

After that, whether we should use markdown for simple tables (yay readable markdown) or use HTML for simple tables (yay homogeneity) is a subjective choice between yays.

[ ¹ Apologies for not realising at the time that there was a way to do the simple markdown tables in both **gh-pages** markdown and GitHub "view" markdown.]